src/Qir/Runtime/lib/QIR/callables.cpp

```diff
@@ -20,10 +20,10 @@ using namespace Microsoft::Quantum;
 ==============================================================================*/
 extern "C"
 {
-    PTuple quantum__rt__tuple_create(int64_t size)
+    PTuple quantum__rt__tuple_create(int64_t size)  // TODO: Use unsigned integer type (breaking change).

```

Comment:
Do you need to take care of this ToDo before merging?

Reply:
I marked as ToDo the things that need a separate consideration (e.g. a synchronous change is needed in more than one repo or a careful change is required in multiple places in one commit).